content/en/data_jobs/databricks.md

```diff
@@ -137,6 +139,39 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 {{< /tabs >}}
 
+### Advanced settings (optional)
```

Comment:
Wondering if this advanced settings section would be better suited in a separated page, to not confuse users in the main setup flow. @warrierr what do you think ?

Reply:
I don't think it needs to be on a separate page, but should be at the bottom of the page in its own header section on the page. 

A few other changes:
1. Can we  rename this "Advanced Configuration"
2. Can we move this to a full section (## instead of ###) and put at the bottom of the page, just above "Further Reading"
3. Can we put the Tag Spans at Runtime under the "Advanced Configuration" section as well

Reply:
updated following @warrierr's recommendations. Also make it explicit that it only applicable to global init when running tasks on a new cluster for each task run.